datafusion/sqllogictest/test_files/group_by.slt

```diff
@@ -4477,13 +4477,19 @@ LIMIT 5
 query ITIPTR rowsort
 SELECT r.*
 FROM sales_global_with_pk as l, sales_global_with_pk as r
-LIMIT 5
+LIMIT 10
 ----
 0 GRC 0 2022-01-01T06:00:00 EUR 30
+0 GRC 0 2022-01-01T06:00:00 EUR 30
+0 GRC 0 2022-01-01T06:00:00 EUR 30
+0 GRC 0 2022-01-01T06:00:00 EUR 30
+0 GRC 0 2022-01-01T06:00:00 EUR 30
 1 FRA 1 2022-01-01T08:00:00 EUR 50
-1 FRA 3 2022-01-02T12:00:00 EUR 200
-1 TUR 2 2022-01-01T11:30:00 TRY 75
-1 TUR 4 2022-01-03T10:00:00 TRY 100
+1 FRA 1 2022-01-01T08:00:00 EUR 50
+1 FRA 1 2022-01-01T08:00:00 EUR 50
+1 FRA 1 2022-01-01T08:00:00 EUR 50
+1 FRA 1 2022-01-01T08:00:00 EUR 50
+
```

Comment:
These test results are updated since we preserve the order of the right table now.